src/functional/iterator/backends/gtfn/gtfn_ir.py

```diff
@@ -0,0 +1,124 @@
+import enum
+from typing import List, Union
+
+from eve import Node
+from eve.traits import SymbolName, SymbolTableTrait
+from eve.type_definitions import StrEnum, SymbolRef
+from functional.iterator.util.sym_validation import validate_symbol_refs
+
+
+@enum.unique
+class GridType(StrEnum):
+    Cartesian = "Cartesian"
+    Unstructured = "unstructured"
```

Comment:
‘Cartesian’ is upper-case, ‘unstructured’ is lower case.

Reply:
It was on purpose. Any preference?

Reply:
Not much of a preference apart from consistency. To better follow the current GT4Py style, I would probably go with a fully upper-case LHS and a lower-case RHS, that is:
```python
@enum.unique
class GridType(StrEnum):
    CARTESIAN = "cartesian"
    UNSTRUCTURED = "unstructured"
```